src/modules/validators/smart-sessions.ts

```diff
@@ -214,6 +220,74 @@ async function getEnableSessionCall(
   }
 }
 
+async function getEnableEmissarySessionCall(
+  chain: Chain,
+  session: Session,
+  accountAddress: Address,
+  provider?: ProviderConfig,
+  _overrideEnable?: SmartSessionEmissaryEnable,
+  _sessionDigest?: Hex,
+) {
+  const { appDomainSeparator, contentsType } =
+    await getSessionAllowedERC7739Content(chain, provider)
+  const allowedERC7739Content = [
+    {
+      appDomainSeparator,
+      contentName: [contentsType],
+    },
+  ]
+  const _sessionData = getSmartSessionData(
+    chain,
+    session,
+    allowedERC7739Content,
+  )
+
+  // Simple emissary session enablement - just call setConfig with basic data
+  const permissionId = getPermissionId(session)
+
+  const configArgs = {
+    sender: SMART_SESSIONS_VALIDATOR_ADDRESS,
+    scope: session.emissary ? Number(session.emissary.scope) : 0,
+    resetPeriod: session.emissary ? Number(session.emissary.resetPeriod) : 2,
+    allocator:
+      session.emissary?.allocator ||
+      '0x0000000000000000000000000000000000000000',
+    permissionId,
+  }
+
+  const enableArgs = {
+    allocatorSig: '0x' as const, // Empty for now
+    userSig: '0x' as const, // Empty for now
+    expires: BigInt(Math.floor(Date.now() / 1000) + 3600),
+    session: {
+      chainDigestIndex: 0,
+      hashesAndChainIds: [
+        {
+          chainId: BigInt(chain.id),
+          sessionDigest:
+            '0x0000000000000000000000000000000000000000000000000000000000000000' as const,
+        },
+      ],
+      sessionToEnable: {
+        sessionValidator: '0x000000000013fdB5234E4E3162a810F54d9f7E98' as const,
+        sessionValidatorInitData: '0x' as const,
+        salt: '0x0000000000000000000000000000000000000000000000000000000000000000' as const,
+        erc1271Policies: [],
+        actions: [],
+      },
+    },
+  }
+
+  return {
+    to: SMART_SESSION_EMISSARY_ADDRESS,
+    data: encodeFunctionData({
+      abi: enableSessionsEmissaryAbi,
```

Comment:
Let's inline the required ABI item here and remove the JSON file to optimize on size.